index.html

```diff
@@ -12539,7 +12533,7 @@ <h2>Keywords</h2>
       <a>node object</a>, <a>value object</a>, <a>graph object</a>, <a>set object</a>, or <a>list object</a>.
       Its value MUST be a <a>string</a>.
       <p>The unaliased <code>@index</code> MAY be used as the value of the <code>@container</code> key within an <a>expanded term definition</a>
-        and as an entry in a <a>expanded term definition</a>, where the value an <a>IRI</a>,
+        and as an entry in an <a>expanded term definition</a>, where the value an <a>IRI</a>,
```

Comment:
```suggestion
        and as an entry in an <a>expanded term definition</a>, where the value is an <a>IRI</a>,
```

Reply:
Fixed in 7ba4fae23cef469dffc691d7f7c1c58b169ba905.